src/main/scala/devices/debug/Debug.scala

```diff
@@ -1254,6 +1254,27 @@ class TLDebugModule(beatBytes: Int)(implicit p: Parameters) extends LazyModule {
       val memRegions = DiplomaticObjectModelAddressing.getOMMemoryRegions("Debug", resourceBindings)
       val cfg = p(DebugModuleParams)
 
+      def getClockRelationships: Seq[OMClockRelationship] = Nil
+
+      def getClocks: Seq[OMClock] = Nil
+
+      def getResets: Seq[OMRTLReset] = Nil
+
+      def getRTLModule(): OMRTLModule = {
+        val omRTLInterface = OMRTLInterface(
+          clocks = getClocks,
+          clockRelationships = getClockRelationships,
+          resets = getResets
+        )
+
+        OMRTLModule(
+          moduleName = name, // TODO Ask Jack, probably need to change the moduleName type to named
+          instanceName = None, // Some(instanceName), // TODO Ask Jack, probably need to change the instanceName type to named
+          hierarchicalId = None, // TODO Ask Jack, probably need to change the hierarchicalId type to named
```

Comment:
This is not ready to be merged until you have a concrete proposal for how this translation is going to work.